doc/install/manual_install.rst

```diff
@@ -28,14 +30,23 @@ Run in your terminal:
 
     $ conda install --channel=conda-forge --name=base mamba
     $ mamba create --override-channels --channel=conda-forge --name=mne mne
-
-This will create a new ``conda`` environment called ``mne`` (you can adjust
-this by passing a different name via ``--name``) and install all
-dependencies into it.
+    $ mamba activate mne
+    $ mamba env config vars set PYTHONNOUSERSITE=1
```

Comment:
Why don't these instructions use `environment.yml`?  If they did, we could set this variable in the environment.yml file (see end of section https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#setting-environment-variables) and not need to bother the user with it. Is there a reason we're not doing that here?

Reply:
We're not using environment.yml for user installations

Reply:
Try using curl on Windows ...

Cheese was incorrect autocorrect, sorry about that! I edited my post above :) it was meant to be "nice". It's not nice (as in, convenient) for users

Reply:
conda-build allows us to set this variable with installation of the mne conda package. But it'd feel a little dirty applying this trick, interfering with the user's environment in a way they probably wouldn't expect. Thoughts on this one?

Reply:
> Try using curl on Windows ...

curl is *built into Windows* nowadays, surely the headaches are over? https://curl.se/windows/microsoft.html

> it'd feel a little dirty applying this trick, interfering with the user's environment in a way they probably wouldn't expect. Thoughts on this one?

I think it's OK because it *only affects the MNE environment* (the envvar disappears when the environment is deactivated).  We should still tell users about it, of course, but then it becomes a "sidebar note for advanced users" instead of something that all users need to pay attention to and take action on.  IMO loading packages from outside the current environment is rarely a good idea (I say this as someone who regularly does it, for what I think are good but corner-case reasons, and it still bites me occasionally).

Reply:
oh, I re-read your post and I see now you're suggesting something different, which is to include that env var *wherever users install `mne`*.  That is different from what I am suggesting, which is to set it in the environment file.  I don't think it's a good idea to include an env var alongside the `mne` conda package itself.